llvm/lib/Transforms/Instrumentation/PGOInstrumentation.cpp

```diff
@@ -1624,7 +1624,9 @@ void PGOUseFunc::setBranchWeights() {
 
     // We have a non-zero Branch BB.
     unsigned Size = BBCountInfo.OutEdges.size();
```

Comment:
Could you also rename `Size` so it is more clear how it is different from `SuccessorCount`?

Reply:
Done.